src/bare-metal/aps/examples/entry.S

```diff
@@ -72,18 +78,20 @@
 .set .Lsctlrval, .Lsctlrval | .L_SCTLR_ELx_I | .L_SCTLR_EL1_SPAN | .L_SCTLR_EL1_RES1
 
 /**
- * This is a generic entry point for an image. It carries out the operations required to prepare the
- * loaded image to be run. Specifically, it zeroes the bss section using registers x25 and above,
- * prepares the stack, enables floating point, and sets up the exception vector. It preserves x0-x3
- * for the Rust entry point, as these may contain boot parameters.
+ * This is a generic entry point for an image. It carries out the
+ * operations required to prepare the loaded image to be run.
+ * Specifically, it zeroes the bss section using registers x25 and
+ * above, prepares the stack, enables floating point, and sets up the
+ * exception vector. It preserves x0-x3 for the Rust entry point, as
+ * these may contain boot parameters.
  */
 // ANCHOR: entry
 .section .init.entry, "ax"
 .global entry
 entry:
     /*
-     * Load and apply the memory management configuration, ready to enable MMU and
-     * caches.
+     * Load and apply the memory management configuration, ready to
```

Comment:
these are spaces to indent, files below have tabs. Should we try to have one style over all files?
This could also be a separate commit

Reply:
Yeah, having a consistent style would be good! Perhaps we could teach `dprint` to do this, but it might not be worth is since we touch these files very very rarely.